docs/docs/migration_notes.md

```diff
@@ -23,15 +23,15 @@ It is now possible to explicitly state a function doesn't perform any state alte
 View functions only generate a `StaticCallInterface` that doesn't include `.call` or `.enqueue` methods. Also, the denomination `static` has been completely removed from the interfaces, in favor of the more familiar `view`
 
 ```diff
-+ let price = PriceFeed::at(asset.oracle).get_price(0).view(&mut context).price;
 - let price = PriceFeed::at(asset.oracle).get_price(0).static_call(&mut context).price;
++ let price = PriceFeed::at(asset.oracle).get_price(0).view(&mut context).price;
```

Comment:
Had to check these in the actual file to clearly see understand what was going on. #diff-ception